src/ontology/mondo-ingest.Makefile

```diff
@@ -160,15 +160,20 @@ mappings: sssom $(ALL_MAPPINGS)
 #################
 # Utils #########
 #################
-# Documentation for this commands in this section is in: `docs/developer/ordo.md`
-
+# Documentation for `report-mapping-annotations` and `update-jinja-sparql-queries`: `docs/developer/ordo.md`
 report-mapping-annotations:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/report_mapping_annotations.py
 
 update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl
```

Comment:
@matentzn Slight issue I'm having with `component-download-%.owl` requirement here.

I don't know if this is just for ICD10CM or all ontologies, but this always downloads, even if the file is always there. I realize that this is probably a good practice in production, but it slows development down a bit. I'm wondering if it's possible to improve these parts of our pipelines by using file hashes (e.g. MD5)? What I mean is, whenever we download or upload a file, we store the hash somewhere, either alongside the file itself, or somewhere within the file name. That way we can check if the hash of the file online is the same as the hash currently on disk. If so, we can skip the download step. What do you think?

For reference, here's the make goal for `component-download-icd10cm.owl`:
```make
component-download-icd10cm.owl: | $(TMPDIR)
	if [ $(MIR) = true ]; then wget $(ICD10CM_URL) -O $(TMPDIR)/icd10cm.tmp.owl && $(ROBOT) remove -i 
```